stable/grafana/values.yaml

```diff
@@ -1,475 +1,148 @@
-## If you'd like to preinstall prometheus dashboard on the same namespace as example, get it from:
+datasources: {}
+#  datasources.yaml:
+#    apiVersion: 1
+#    datasources:
+#    - name: Prometheus
+#      type: prometheus
+#      url: http://prometheus-prometheus-server
+#      access: proxy
+#      isDefault: true
+
+## Configure grafana dashboard providers
+## ref: http://docs.grafana.org/administration/provisioning/#dashboards
 ##
-## https://grafana.net/dashboards/2
+dashboardProviders: {}
+#  dashboardproviders.yaml:
+#    apiVersion: 1
+#    providers:
+#    - name: 'default'
+#      orgId: 1
+#      folder: ''
+#      type: file
+#      disableDeletion: false
+#      editable: true
+#      options:
+#        path: /var/lib/grafana/dashboards
+
+## Configure grafana dashboard to import
+## ref: https://grafana.com/dashboards
 ##
-## and add it below.
+dashboards: []
+#  - name: some-dashboard
+#    json: |
+#      $RAW_JSON
+#  - name: prometheus-stats
+#    gnetId: 2
+#    revision: 2
+#    datasource: Prometheus
+#  - name: node-exporter-service-metrics
+#    gnetId: 405
+#    revision: 6
+#    datasource: Prometheus
+#  - name: kubernetes-cluster-monitoring-via-prometheus
+#    gnetId: 1621
+#    datasource: Prometheus
+#  - name: kubernetes-node-exporter-full
+#    gnetId: 3320
+#    datasource: Prometheus
+
+## Grafana's primary configuration
+## NOTE: values in map will be converted to ini format
+## ref: http://docs.grafana.org/installation/configuration/
 ##
-dashboardImports:
-  ## If true, a Job will be created to import dashboards
-  ## Default: false
-  ##
-  enabled: false
-
-  ## How long should it take to commit failure
-  ## Default: 30
-  ##
-  activeDeadlineSeconds: 30
-
-  ## Docker image to run import process
-  ## Default: jgoclawski/wget:1.0
-  ##
-  image: jgoclawski/wget:1.0
-
-  ## Grafana dashboard files to import as ConfigMap entries
-  # files: {}
-
-  ## Grafana dashboard URL to import (this should be a JSON file, not the HTML page)
-  # Example:
-  # dashboards:
-  #   Prometheus_Stats.json: https://grafana.com/api/dashboards/2/revisions/2/download
-  #   Docker_Dashboard.json: https://grafana.com/api/dashboards/179/revisions/5/download
-  #   MyPrivateDashboard.json: http://example.com/dashboard.json
-  # dashboards: {}
-
-  ## Specify the Job policy
-  ## Default: Never
-  ##
-  restartPolicy: Never
-
-  ## Specify Pod Backoff failure policy
-  ## Default: 1
-  ##
-  backoffLimit: 1
-
-  ## Specify when Helm should run the import job
-  ## Default: post-install,post-upgrade
-  ##
-  hook: post-install,post-upgrade
+grafana.ini:
+  default:
+    instance_name: ${HOSTNAME}
+  paths:
+    data: /var/lib/grafana/data
+    logs: /var/log/grafana
+    plugins: /var/lib/grafana/plugins
+  analytics:
+    check_for_updates: true
+  log:
+    mode: console
+    level: info
+  dashboards.json:
+    enabled: true
+    path: /var/lib/grafana/dashboards
```

Comment:
In G5, `dashboards.json` is deprecated. Since this change is not backwards compatible with G4, could we just nuke the section, for clarity?

Reply:
@DaytonG Good catch. Fixed